gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -590,7 +590,7 @@ export class AdvancedDynamicTexture extends DynamicTexture {
         }
 
         this._pointerMoveObserver = scene.onPrePointerObservable.add((pi, state) => {
-            if (scene!.isPointerCaptured((<PointerEvent>(pi.event)).pointerId)) {
+            if (scene!.isPointerCaptured((<PointerEvent>(pi.event)).pointerId) && pi.type !== PointerEventTypes.POINTERUP) {

```

Comment:
See PR description, let me know if this change is the right one.

Reply:
What about adding && this._lastControlOver[pointerId] so we make sure to only transmit the event if there is a potential receiver?

Reply:
I added 

            if(!this._lastControlOver[(<PointerEvent>(pi.event)).pointerId]){
                return;
            }

if that is what you mean't. It broke playground.babylonjs.com#S7L7FE as there is no this._lastControlOver[pointerId] ever causing it to never get focus.

Reply:
Another potential fix is to change when isPointerCaptured is true. scene!.isPointerCaptured is set to true after pointerdown is fired and set to false after prePointerObservables. isPointerCaptured was created as part of this pr https://github.com/BabylonJS/Babylon.js/issues/4236 . If i move the  isPointerCaptured  setting to false before prePoitnerObservable then it still works and I'd expect scene!.isPointerCaptured to be false here.

Reply:
Well I meant:
if (scene!.isPointerCaptured((<PointerEvent>(pi.event)).pointerId) 
&& (pi.type !== PointerEventTypes.POINTERUP) || !this._lastControlOver[(<PointerEvent>(pi.event)).pointerId]) {